.size-snapshot.json

```diff
@@ -430,16 +430,16 @@
     }
   },
   "shared-components/alert/style.js": {
-    "bundled": 53374,
-    "minified": 52524,
-    "gzipped": 5533,
+    "bundled": 35497,
+    "minified": 34532,
+    "gzipped": 5262,
```

Comment:
Interestingly, removing the `css` SerializedStyles in `styled` components appears to bloat the size a bit. Removing some of those instances accounts for the change in this PR.